sql/localsql/certifier/db.go

```diff
@@ -12,26 +12,26 @@ type PoetCert struct {
 	Signature []byte
 }
 
-func AddCertificate(db sql.Executor, nodeID types.NodeID, cert PoetCert, poetUrl string) error {
+func AddCertificate(db sql.Executor, nodeID types.NodeID, cert PoetCert, cerifierID []byte) error {
```

Comment:
We switched from poet ID to URL so we can more easily log and find the correct poet for the servers we submitted to, why change it back again?

Reply:
I changed here to certifier ID as a single certificate is valid for the pair (nodeID, certifier), not (nodeID, poet), because many poets can share the same certificate. Both certifier ID and URL could be used, I picked the ID as this will never change (the certificates would become invalid then). The URL, on the other hand, could change and the certificates would still be valid.

Reply:
I'm not sure about this. Yes, certificates have an ID but they are usually bound to some domain. A certificate usually isn't valid for arbitrary domains even if the new domain points to the same service/key.

Also I am unsure about this feature - why would someone run a certifier service and then change its domain but nothing else about it?

Reply:
It's the certifier ID (its public key), not the certificate ID. The certificate is valid for a pair of (node ID and certifier pubkey), not (node ID and certifier domain). I'm not sure why would somebody change the domain (maybe forgot to pay and lost it ?), but that's fine and the certificates issued in the past **are still valid** from the poet's point of view. I see no reason to limit this on the node side.